### DISCUSSION TITLE
Page does not render after including specific code in markup

### DISCUSSION THREAD

**Discussion by frauzufall** [requarks/wiki#3566] [Error / Bug Report]
**Describe the bug**
When including specific code in a markup page, the resulting HTML page is empty.

**To Reproduce**
1. Create new page
2. Set markdown content to:
```
  - `{,}`
```
3. View page
4. See white page. The browser console displays `DOMException: "String contains an invalid character" app.3f0ec7bfc994316d5963.js:102:11676`

**Expected behavior**
See not white page.
Note: Also the preview does not render this code snippet but the remaining page would be rendered at least in the preview.

**Host Info (please complete the following information):**
 - OS: Ubuntu 18.10
 - Wiki.js version: 2.0.0-beta.241
 - Database engine: postgres
- Client setups: Ubuntu (Firefox), MacOS (Safari)

**Additional context**
In the markdown, the comma as well as the dash are important. Note that additional text at the end of the line restores functionality.

This works: 
```
  - `{,}`A
```
This works partially (does not render the code section, but does not break the page):
```
  - `{A}`
```

**Comment by SharkProgramming** [comment]
I have the same issue with the combination of two characters { } with text inside

![issue2](https://user-images.githubusercontent.com/33226400/68190404-9d00a100-ffad-11e9-8447-0781365c164f.png)

**Comment by Smankusors** [comment]
Is this issue still happens with Wiki.js 2.4.x?
